panther_analysis_tool/main.py

```diff
@@ -815,6 +824,29 @@ def classify_analysis(
             # validate the particular analysis type schema
             analysis_schema = SCHEMAS[analysis_type]
             keys = list(analysis_schema.schema.keys())
+            # Temporarily set schema definition to ignore validation
+            if ignore_logtypes_validation:
+                if "LogTypes" in analysis_spec:
+                    # Workaround since this is stored as "Or('LogTypes', 'ScheduledQueries'): [str]"
+                    for each_key in analysis_schema.schema.keys():
+                        if str(each_key) == "Or('LogTypes', 'ScheduledQueries')":
+                            tmp_logtypes_key = each_key
+                            break
+                    tmp_logtypes = analysis_schema.schema[tmp_logtypes_key]
+                    analysis_schema.schema[tmp_logtypes_key] = [str]
+            if ignore_resourcetypes_validation:
+                if "ResourceTypes" in analysis_spec:
+                    tmp_resourcetypes = analysis_schema.schema["ResourceTypes"]
+                    analysis_schema.schema["ResourceTypes"] = [str]
```

Comment:
Is there a specific request/reason we would want to support ignoring this validation? Just wondering if this adds complexity when on the backend it will fail / there isn't a way to skip the validation.

Reply:
This was namely added for the case of custom logtypes/schemas